setup.py

```diff
@@ -29,8 +46,10 @@ def build_package(version):
     setuptools.setup(
         name="smdebug",
         version=version,
+        long_description="\n".join(DOCLINES[2:]),
+        long_description_content_type="text/x-rst",
         author="AWS DeepLearning Team",
-        description="Automated debugging for machine learning",
+        description=DOCLINES[0],
```

Comment:
What are we changing this to? Someone might change the README later and unintentionally mess up the description. Might be best to just leave the description hard-coded.

Reply:
this change was done on Amol's request. @leleamol